config/settings.yml

```diff
@@ -5,7 +5,7 @@ purl_conn_timeout: 2
 purl_feedback_email: ''
 valid_purl_url: <%= /https?:\/\/purl\.stanford\.edu\/*/ %>
 stacks_url: 'https://stacks.stanford.edu'
-iiif_stacks_url: 'https://stacks.stanford.edu'
+iiif_stacks_url: 'https://stacks.stanford.edu' # unused???
```

Comment:
done.   This also made me think:  we're starting to get better at documenting why we include gems in a Gemfile.  Should we not be better about documenting un-obvious things in a settings file?   Like "valid_purl_url" and "embed_iframe_url" -- those aren't obvious to me.  (might the latter be better named "iframe_url_prefix" or something?)  If you agree, maybe we should 
* make a ticket for this?
* watch for this on PRs in the future